rts/Game/Game.cpp

```diff
@@ -2147,13 +2053,13 @@ void CGame::ActionReceived(const Action& action, int playerID)
 	}
 }
 
-bool CGame::ActionPressed(unsigned int key, const Action& action, bool isRepeat)
+bool CGame::ActionPressed(unsigned int keyCode, unsigned int scanCode, const Action& action, bool isRepeat)
 {
 	const IUnsyncedActionExecutor* executor = unsyncedGameCommands->GetActionExecutor(action.command);
 
 	if (executor != nullptr) {
 		// an executor for that action was found
-		if (executor->ExecuteAction(UnsyncedAction(action, key, isRepeat)))
+		if (executor->ExecuteAction(UnsyncedAction(action, keyCode, isRepeat)))
```

Comment:
No scancode here? (sorry if some comments are dumb :D)

Reply:
We could propagate scancodes to actions if we wanted to, I intended to keep the surface area minimal, but that can be done. Ideally this should be done when the need arises for actions to be aware of scancodes but I don't mind extending this PR.